aip/general/0192.md

```diff
@@ -108,6 +108,12 @@ colloquial use that avoiding it would obscure the reference (example: [IBM][]).
 Comments **should** spell and capitalize trademarked names consistent with the
 trademark owner's current branding.
 
+### Deprecations
+
+To deprecate a service or RPC, the `deprecated` option **must** be set to
```

Comment:
I think this should apply to deprecating fields as well. WDYT?

Reply:
I could go either way, but the generators don't do anything with them ATM. @lukesneeringer WDYT?

Reply:
Let's do this universally on anything.